### PR TITLE
fix: remove line_number from RulebookProcessLog

### PR DESCRIPTION
**Pull request by jshimkus-rh**
Constantly querying during log writing is a severe bottleneck and the line number provides little actual value.

Also changes log processing flush default to every 100 entries rather than after each entry.
This further enhances the log writing performance.

https://issues.redhat.com/browse/AAP-21897
https://issues.redhat.com/browse/AAP-19845
https://issues.redhat.com/browse/AAP-21033
https://issues.redhat.com/browse/AAP-20821
https://issues.redhat.com/browse/AAP-24186

### CODE REVIEW

File: tools/docker/docker-compose-dev-redis-tls.yaml
Comment (mkanoor) [inline tools/docker/docker-compose-dev-redis-tls.yaml:36]: @jshimkus-rh Is this related to the logging issue?

Comment (jshimkus-rh) [reply tools/docker/docker-compose-dev-redis-tls.yaml:36]: The FLUSH_AFTER definitely as it provides a way to manipulate the value for performance characterization.

The DEFAULT_QUEUE_TIMEOUT a bit less directly but being able to easily manipulate timeouts was valuable in preventing the system from going belly up while investigating the origins.

File: src/aap_eda/core/migrations/0040_remove_rulebookprocesslog_line_number.py
Comment (Alex-Izquierdo) [inline src/aap_eda/core/migrations/0040_remove_rulebookprocesslog_line_number.py:12]: @jshimkus-rh Have you check with @lgalis if there is an impact on the UI?

Comment (jshimkus-rh) [reply src/aap_eda/core/migrations/0040_remove_rulebookprocesslog_line_number.py:12]: Have checked with others and we're almost certainly ok.
But, as I said in slack, if people want to wait on the merge until @lgalis is back and can weigh in that's fine by me.  But I'd like reviews to take place under the assumption that it'll be fine so we're ready to merge.

Comment (Alex-Izquierdo) [reply src/aap_eda/core/migrations/0040_remove_rulebookprocesslog_line_number.py:12]: LGTM, I'm not going to approve until be sure it can be merged :)

**Review by Alex-Izquierdo — DISMISSED** [review]

**Review by Alex-Izquierdo — APPROVED** [review]

File: tools/docker/docker-compose-dev-redis-tls.yaml
Comment (Alex-Izquierdo) [reply tools/docker/docker-compose-dev-redis-tls.yaml:37]: @mkanoor is you question solved? Can we merge?

**Pull request merged**
